packages/flutter/lib/src/widgets/sensitive_content.dart

```diff
@@ -0,0 +1,225 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/services.dart' show ContentSensitivity, SensitiveContentService;
+
+import 'framework.dart';
+
+/// Data structure used to track the number of [SensitiveContent] widgets with
+/// each of the different [ContentSensitivity] levels set in a particular
+/// Flutter view.
```

Comment:
Is it clear to users what a Flutter view is? I want to make sure we have some standard nomenclature here.

I left another comment asking about multiwindow. Does this mean that each window in a multi-window Flutter app would have its own viewId, and that the sensitivity should be handled separately for each? Or, how about the scenario where you have an Android app with add-to-app Flutter instances in two separate places?